pbsmetrics/config/metrics.go

```diff
@@ -284,3 +308,108 @@ func (me *DummyMetricsEngine) RecordTimeoutNotice(success bool) {
 // RecordReq as a noop
 func (me *DummyMetricsEngine) RecordTCFReq(version pbsmetrics.TCFVersionValue) {
 }
+
+// MockMetricsEngine will have fields to update via the MetricsEngine interface methods
+type MockMetricsData struct {
```

Comment:
Please use the existing `MetricsEngineMock` instead of creating a new one.

Reply:
:laughing: I just used `MetricsEngineMock` in `TestCallRecordAdapterConnections` above and completely forgot about it. Jesus, I'm sorry. Fixed and removed.